doc/classes/SpinBox.xml

```diff
@@ -41,7 +41,7 @@
 			<return type="LineEdit" />
 			<description>
 				Returns the [LineEdit] instance from this [SpinBox]. You can use it to access properties and methods of [LineEdit].
-				[b]Warning:[/b] This is a required internal node, removing and freeing it may cause a crash. If you wish to hide it or any of its children, use their [member CanvasItem.visible] property.
+				[b]Warning:[/b] This is a required internal node, removing and freeing it may cause a crash. The internal [LineEdit] is not intended to be changed directly as [SpinBox] controls it, manually change it may cause some appearance issue. If you wish to hide it or any of its children, use their [member CanvasItem.visible] property.
```

Comment:
```suggestion
				[b]Warning:[/b] This is a required internal node, removing and freeing it may cause a crash. It is managed by this [SpinBox] and is not intended to be changed directly, manually changing it may cause some appearance issues. If you wish to hide it or any of its children, use their [member CanvasItem.visible] property.
```
Some suggestions for rewording, at least some grammar notes

Reply:
Thanks !